src/harness/BenchmarkDotNet.Extensions/ValuesGenerator.cs

```diff
@@ -114,5 +147,23 @@ private static string GenerateRandomString(Random random, int minLength, int max
 
             return builder.ToString();
         }
+
+        private static T[] AllocateAlignedArray<T>(int size, bool aligned, out GCHandle pinnedArrayHandle)
```

Comment:
nit: The method name (`AllocateAlignedArray`) is misleading when you are calling it to allocate an un-aligned array.